e2e-tests/non-voting/run

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+check_cr_config() {
```

Comment:
you can use `spinup_psmdb` function

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/723/commits/78b976cdc932c4381c8343c0811c45e8feddb755